app/src/main/java/app/revanced/integrations/patches/LithoThemePatch.java

```diff
@@ -22,13 +22,25 @@ public class LithoThemePatch {
 
     // Used by app.revanced.patches.youtube.layout.theme.patch.LithoThemePatch
     public static int applyLithoTheme(int originalValue) {
-        var isDarkTheme = ThemeHelper.isDarkTheme();
-        
-        if ((isDarkTheme && anyEquals(originalValue, DARKCONSTANTS)) || (!isDarkTheme  && anyEquals(originalValue, WHITECONSTANTS)))
-                return 0;
+        if (ThemeHelper.isDarkTheme()) {
+            if (anyEquals(originalValue, DARKCONSTANTS))
+                return getColor("yt_black1");
+        } else if (anyEquals(originalValue, WHITECONSTANTS))
+            return getColor("yt_white1");
+
         return originalValue;
     }
 
+    private static int getColor(String colorName) {
+        return context.getColor(
```

Comment:
This should not be called this often. Instead store the color inside a field. I believe a getter also should be used instead accessing the field `context`. Please add a field for the return value of this method. Check if it is set every time the method is called, if not, set it, if it is set, return it.

Reply:
I don't know how to use getters and setters. I've never used them.

If you want, help me to fix this.

Reply:
EDIT: Based on what I understand from your comment, the new commit should be enough.

Reply:
How are you sure context is set when the fields are set on static class initialization?

Reply:
Please do as previously suggested:

> Check if it is set every time the method is called, if not, set it, if it is set, return it.

Reply:
Ok, thanks for clarifying this request further. I wasn't sure if I applied all the requests.

Reply:
Please check on [this](https://github.com/revanced/revanced-integrations/pull/239#discussion_r1054825073) as well.

Reply:
Ok...so this is what I do:

1) Changed the static context getter to the method "getContext()", and created a dedicated field for the latter with a null check inside the getColor() method.

2) Moved the color fields setter to applyLithoTheme() method, and placed them inside their unique theme condition. In this way there will be a single field setter, based on the current theme.

3) Added field value checks, to avoid multiple getColor() method calls.

And I shouldn't have forgotten anything.